pycalphad/core/sympydiff_utils.py

```diff
@@ -143,18 +143,17 @@ class f(Function):
         restup.append(AutowrapFunction(args, Eq(y[i], f(*args_with_indices))))
     if include_grad or include_hess:
         diffargs = (inp_nobroadcast, params)
-        # Replacing zoo's is necessary because sympy's CCodePrinter doesn't handle them
```

Comment:
Does this set a new minimum required SymPy version?

Reply:
No.